include/inexor/vulkan-renderer/wrapper/device.hpp

```diff
@@ -13,90 +13,86 @@ namespace inexor::vulkan_renderer::wrapper {
 /// @brief A RAII wrapper for VkDevice, VkPhysicalDevice and VkQueues.
 class Device {
 private:
-    VkDevice device;
-    VkPhysicalDevice graphics_card;
+    VkDevice m_device;
+    VkPhysicalDevice m_graphics_card;
     VmaAllocator m_allocator;
 
     // TODO(): Create a queue wrapper
-    VkQueue graphics_queue;
-    VkQueue present_queue;
-    VkQueue transfer_queue;
-    VkSurfaceKHR surface;
+    VkQueue m_graphics_queue;
+    VkQueue m_present_queue;
+    VkQueue m_transfer_queue;
+    VkSurfaceKHR m_surface;
 
-    std::uint32_t present_queue_family_index;
-    std::uint32_t graphics_queue_family_index;
-    std::uint32_t transfer_queue_family_index;
+    std::uint32_t m_present_queue_family_index;
+    std::uint32_t m_graphics_queue_family_index;
+    std::uint32_t m_transfer_queue_family_index;
 
     // The debug marker extension is not part of the core,
     // so function pointers need to be loaded manually.
-    PFN_vkDebugMarkerSetObjectTagEXT vk_debug_marker_set_object_tag;
-    PFN_vkDebugMarkerSetObjectNameEXT vk_debug_marker_set_object_name;
-    PFN_vkCmdDebugMarkerBeginEXT vk_cmd_debug_marker_begin;
-    PFN_vkCmdDebugMarkerEndEXT vk_cmd_debug_marker_end;
-    PFN_vkCmdDebugMarkerInsertEXT vk_cmd_debug_marker_insert;
-    PFN_vkSetDebugUtilsObjectNameEXT vk_set_debug_utils_object_name;
+    PFN_vkDebugMarkerSetObjectTagEXT m_vk_debug_marker_set_object_tag;
```

Comment:
TODO for me: Implement a `VulkanObject` base class which contains Vulkan debug markers and inherit from it.